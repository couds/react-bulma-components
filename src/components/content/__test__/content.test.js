import React from 'react';
import renderer from 'react-test-renderer';
import Content from '..';

describe('Content component', () => {
  it('Should have content classname', () => {
    const component = renderer.create(
      <Content>
        <p className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </p>
      </Content>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
