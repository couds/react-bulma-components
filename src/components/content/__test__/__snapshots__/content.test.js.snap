// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component Should have content classname 1`] = `
<div
  className="content"
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;

exports[`Content component Should have content classname with normal size 1`] = `
<div
  className="content is-normal"
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;
